crates/nu-command/src/commands/network/fetch.rs

```diff
@@ -361,9 +369,10 @@ async fn helper(
 // Only panics if the user agent is invalid but we define it statically so either
 // it always or never fails
 #[allow(clippy::unwrap_used)]
-fn http_client() -> reqwest::Client {
+fn http_client(allow_insecure: bool) -> reqwest::Client {
     reqwest::Client::builder()
         .user_agent("nushell")
+        .danger_accept_invalid_certs(allow_insecure)
         .build()
```

Comment:
Instead of unwrap use expect with a message explaining why it shouldn't panic. This will help for future debugging